index.js

```diff
@@ -0,0 +1,103 @@
+const titleInput = document.querySelector('.titleInput');
+const bookButton = document.querySelector('.book-button');
+const bookList = document.querySelector('.book-list');
+
+const authorInput = document.getElementById('authorInput');
+const pageInput = document.getElementById('pageInput');
+const readCheckbox = document.getElementById('readCheckbox');
+const form = document.getElementById('form');
+
+const myLibrary = [];
+
+function Book(title = '', author = '', pages = '', read = false) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+function createBookCard(book, index) {
+  const bookDiv = document.createElement('div');
+  bookDiv.classList.add('book');
+  // To do list
+  const newBook = document.createElement('li');
+  newBook.textContent = `Book Name: ${book.title} `;
+  newBook.classList.add('book-item');
+
+  bookDiv.appendChild(newBook);
+
+  const author = document.createElement('li');
+  author.textContent = `Author : ${book.author} `;
+  author.classList.add('book-item');
+
+  bookDiv.appendChild(author);
+
+
+  const page = document.createElement('li');
+  page.textContent = `Pages: ${book.pages} `;
+  page.classList.add('book-item');
+
+  bookDiv.appendChild(page);
+
+  // Check mark Button
+  const completeButton = document.createElement('button');
+  completeButton.setAttribute('data-index', index);
+  completeButton.innerText = book.read ? 'UnRead' : 'Read';
+  completeButton.classList.add('complete-btn');
+  bookDiv.appendChild(completeButton);
+
+  // Trash button
+  const trashButton = document.createElement('button');
+  trashButton.setAttribute('data-index', index);
+  trashButton.innerHTML = '<i class = "fa fa-trash"></i>';
+  trashButton.classList.add('trash-btn');
+  bookDiv.appendChild(trashButton);
+  // Append to list
+  bookList.appendChild(bookDiv);
+
+  /* eslint-disable no-use-before-define */
+  completeButton.addEventListener('click', checkBox);
+  trashButton.addEventListener('click', deleteBook);
+  /* eslint-enable no-use-before-define */
```

Comment:
Always strive to understand linter warnings and errors. This will ensure that you understand and probably find a way to fix the error, and not just disable it.

The `no-use-before-define` rule is eslint's way of telling you not to invoke a function before it is declared. In other words, moving the `checkBox` and `deleteBook` function up before the `createBookCard` function will fix this warning/error.